src/pybind/mgr/dashboard_v2/frontend/src/app/core/auth/login/login.component.html

```diff
@@ -42,6 +42,17 @@ <h1 translate>Welcome to ceph!</h1>
                *ngIf="(loginForm.submitted || password.dirty) && password.invalid">Password is required</div>
         </div>
 
+        <!-- Stay signed in -->
+        <div class="checkbox">
+          <label translate>
+            <input id="stay_signed_in"
+                 name="stay_signed_in"
```

Comment:
Is this the correct indentation when we have one attribute per line?

Reply:
Thx for the tip. Fixed.